aspnet-core/xml/ns-Microsoft.AspNetCore.Authentication.xml

```diff
@@ -1,6 +1,5 @@
 <Namespace Name="Microsoft.AspNetCore.Authentication">
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Contains types that enable support for authentication.</summary>
```

Comment:
This namespace is pretty important. It should have some more information. See some of the longer summaries at https://docs.microsoft.com/en-us/dotnet/api/?view=netcore-3.1

Also link to https://docs.microsoft.com/aspnet/core/security/authentication in the remarks:

`<remarks>For more information, see <see href="https://docs.microsoft.com/aspnet/core/security/authentication">Overview of ASP.NET Core authentication</see>.</remarks>`